SConstruct

```diff
@@ -70,7 +70,7 @@ def get_version():
         return version_file.read().rstrip()
 
 API_VERSION_MAJOR = "1"
-API_VERSION_MINOR = "2"
+API_VERSION_MINOR = "3"
 API_VERSION_FIX = "2"
```

Comment:
Shouldn't the fix number be reset to 0 when bumping the minor?

Reply:
im not sure, should it?